package.json

```diff
@@ -109,8 +108,9 @@
     "svgo": "2.8.0",
     "svgo-loader": "3.0.1",
     "vue-router": "4.1.5",
+    "vue3-jest": "^27.0.0-alpha.1",
```

Comment:
Can we check why we need this one? I don't like it's alpha and I also see we have `@vue/vue3-jest` already. I think we should be using that one.

Reply:
For sure, I will leave `@vue/vue3-jest`, but in peerDependencies there is ` "@vue/vue3-jest": "^27.0.0-alpha.3",`. In master too.